MdeModulePkg/Test/MdeModulePkgHostTest.dsc

```diff
@@ -83,10 +83,7 @@
       UefiBootServicesTableLib|MdePkg/Library/UefiBootServicesTableLib/UefiBootServicesTableLib.inf
   }
 
-  MdeModulePkg/Bus/Pci/NvmExpressDxe/UnitTest/MediaSanitizeUnitTestHost.inf {
-    <LibraryClasses>
-      NvmExpressDxe|MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressDxe.inf
-  }
+  MdeModulePkg/Bus/Pci/NvmExpressDxe/UnitTest/MediaSanitizeUnitTestHost.inf  # MU_CHANGE - Invalid Library override

```

Comment:
It looks like this is not in [edk2/master/MdeModulePkg/Test/MdeModulePkgHostTest.dsc](https://raw.githubusercontent.com/tianocore/edk2/master/MdeModulePkg/Test/MdeModulePkgHostTest.dsc). The `MU_CHANGE` comment is needed, but it could just mention adding the test since the invalid library class override is irrelevant now.

Reply:
Updated comment - thanks!